app/views/pages/home.html.erb

```diff
@@ -0,0 +1,14 @@
+<div class="fluid-container text-center cover">
```

Comment:
The bootstrap class is [container-fluid, not fluid-container](http://getbootstrap.com/css/#overview-container). (If you inspect your page in Chrome, you can see that fluid-container isn't applying any styles to it